doc/dnf5_plugins/index.rst

```diff
@@ -8,6 +8,7 @@ DNF5 Plugins
 
     builddep.8
     copr.8
+    needs_restarting.8
```

Comment:
We still need to deploy the new man page, it is missing in the `doc/conf.py.in`, `doc/CMakeLists.txt` and `dnf5.spec` files.

Reply:
And please also add a reference to new plugin command into `dnf5.8.rst`.